tools/swift-def-to-yaml-converter/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+add_swift_host_tool(swift-def-to-yaml-converter
+  swift-def-to-yaml-converter.cpp
+  SWIFT_COMPONENT tools
+)
+
+target_link_libraries(swift-def-to-yaml-converter PRIVATE
+  swiftLocalization
+  swiftAST)
```

Comment:
I don't think swiftAST is needed so long as you're just #including the .def files, and if that's the case removing it would speed up the build